docs/contributing/index.md

```diff
@@ -1,310 +1,153 @@
 
-    git pull
-    ```
+Documentation
+:   "If it's not documented, it's broken."
+    See {doc}`documentation/index`.
 
-1.  Create a new branch from the development branch.
+Plone API
+:    API methods for Plone functionality.
+    See {doc}`../plone.api/contribute/index`.
 
-    ```shell
-    git checkout -b <new_branch>
-    ```
+Plone REST API
+:   A RESTful API for Plone.
+    See {doc}`plone.restapi/docs/source/contributing/index`.
 
-1.  Make edits to files in `docs/<external_package>` using your favorite editor, and save, preview, and test.
-    You must run and pass the builds `html` and `linkcheck` without causing new errors.
+Volto
+:   Plone 6 default frontend.
+    See {doc}`../volto/developer-guidelines/contributing`.
 
-    ```shell
-    # Optionally clean the builds to avoid cache issues.
-    # Note that for the external packages' documentation only,
-    # we use "docs-" as a prefix for make targets to avoid a conflicts.
-    make docs-clean
-    make docs-html
-    make docs-linkcheck
-    ```
+`plone.app.contentlisting`
+:   Facilitates working with Plone content objects.
+    See https://github.com/plone/plone.app.contenttypes
 
-1.  Back in `submodules/<external_package>`, commit and push your changes to the remote.
+`plone.app.contenttypes`
+:   Provides default content types for Plone.
+    See https://github.com/plone/plone.app.contenttypes
 
-    ```shell
-    git add <files>
-    git commit -m "My commit message"
-    git push
-    ```
+`plone.app.event`
+:   A calendar framework for Plone.
+    See https://github.com/plone/plone.app.event
 
-1.  Now return to the project root directory, and update the submodule to point to the commit you just made, and push your changes to the remote repository.
+`plone.app.multilingual`
+:   The default solution to create multilingual content in a Plone site.
+    See https://github.com/plone/plone.app.multilingual
 
-    ```shell
-    cd ../..
-
-    # for plone.api
-    git add submodules/plone.api
-    git commit -m "Update submodules/plone.api tip"
-
-    # for plone.restapi
-    git add submodules/plone.restapi
-    git commit -m "Update submodules/plone.restapi tip"
-
-    # for Volto
-    git add submodules/volto
-    git commit -m "Update submodules/volto tip"
-
-    git push
-    ```
-
-1.  Visit the GitHub `plone/<external_package>` repository, and create a pull request against the development branch.
-1.  Members who subscribe to the `plone/<external_package>` repository will receive a notification and review your request.
-1.  Request a review from other team members.
-
-
-(contributing-code-of-conduct-label)=
-
-## Code of Conduct
-
-The Plone Foundation has published a [Code of Conduct](https://plone.org/foundation/materials/foundation-resolutions/code-of-conduct).
-All contributors to the Plone Documentation follow the Code of Conduct.
+```{todo}
+Add other important Plone packages to both the above list and the `toctree`.
```

Comment:
Overall I am not sure if we want to start listing packages here at all.

Reply:
This is totally necessary and I feel strongly about it. How do we direct new contributors to the correct contributing information? There should be one for the organization for global stuff, and one each for any deviations from the organization.

There are zero new contributors to Classic UI, Zope, `plone.api`, and Plone Core because no one new knows, except us ancient neckbeards, where to look for all the various bits. If we don't document this structure, it will die when we retire, die, or move on to other shiny things.

Reply:
I agree, and that's why I used the qualifier "important". But I need help with deciding what is "important" to list. Maybe "selected" is a better qualifier?

Reply:
Fixed in https://github.com/plone/documentation/pull/1478/commits/ff6115527c03f77b61c8a51f11bb738d8da4edb2. There is also introductory text above the list for additional context and clarification.